docs/content/mobile/android/local-deploy.md

```diff
@@ -9,15 +9,21 @@ Before reading this document, make sure you are familiar with:
 
 ## Prepare your project to connect to a local deployment {#prepare}
 
-To connect your Android device to a local deployment, you need to prepare its configuration. Follow the steps below to ensure any local deployment you launch via the Unity Editor is correctly set up.
+To connect your Android device to a local deployment, you need to prepare its configuration. Follow the steps below to ensure any local deployment you launch via your Unity Editor is correctly set up.
 
-1. Open your project in the Unity Editor.
+1. Open your project in your Unity Editor.
 1. Navigate to **SpatialOS** > **GDK Tools configuration** to open the configuration window.
 1. In the **Runtime IP for local deployment** field, enter your local machine's IP address. (You can find how to do this on the [Lifehacker website](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address).)
 1. Select **Save** and close the window.
 
+Create and add a MonoBehaviour script to your Android client-worker GameObject. To do this:
+
+1. If your project does not already contain one, create a MonoBehaviour script which inherits from the [`MobileWorkerConnector`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs) and include the functionality you want.
+1. In your Unity Editor, add the MonoBehaviour script to your Android client-worker GameObject.
```

Comment:
We talk about these worker GameObjects - do we have this documented anywhere that we can link to?

Reply:
This will be addressed in UTY-1794, but is out of scope of this bugfix.